dimod/binary_quadratic_model.py

```diff
@@ -120,12 +120,12 @@ class BinaryQuadraticModel(abc.Sized, abc.Container, abc.Iterable):
         >>> bqm = dimod.BinaryQuadraticModel({0: 1, 1: -1, 2: .5},
         ...                                  {(0, 1): .5, (1, 2): 1.5},
         ...                                  1.4,
-        ...                                  dimod.SPIN)
+        ...                                  dimod.Vartype.SPIN)
```

Comment:
why this change? SPIN and BINARY are in the top-level namespace

Reply:
Yes, and I wanted to show that with the two consecutive examples using both supported formats rather than just the one